package.json

```diff
@@ -4,26 +4,22 @@
   "description": "Generates an AEM Starter Kit",
   "license": "SEE LICENSE IN LICENSE.txt",
   "scripts": {
-    "build": "run-s build:theme build:clientlib build:template build:package",
+    "build": "run-s build:theme-compiled build:template build:package",
     "deploy": "curl -u admin:admin -F 'file=@starterkit.zip' http://localhost:4502/conf/global/starterkits.import.html",
-
-    "build:theme": "cd site.theme && npm install && npm run prod && cd ..",
-    "build:clientlib": "clientlib --verbose",
+    "build:theme-compiled": "cd site.theme && npm install && npm run prod && cd ..",
```

Comment:
I would stick to "build:theme" name. The result of building theme is the compiled theme but it doesn't have to be included in the name of npm script.